.env.template

```diff
@@ -102,10 +102,6 @@ CUSTOM_SEARCH_ENGINE_ID=your-custom-search-engine-id
 # USE_MAC_OS_TTS - Use Mac OS TTS or not (Default: False)
 USE_MAC_OS_TTS=False
 
-### ELEVENLABS
```

Comment:
Is the removal of these ELEVENLABS elements intentional?

Reply:
Sorry it was not intentional.